src/components/InfoField.js

```diff
@@ -1,23 +1,27 @@
 import React from 'react'
-import { textStyle, useTheme, GU } from '@aragon/ui'
+import { Field, GU } from '@aragon/ui'
 import PropTypes from 'prop-types'
 
 function InfoField({ label, children, ...props }) {
-  const theme = useTheme()
-
   return (
-    <div {...props}>
-      <h3
-        css={`
-          margin-bottom: ${0.75 * GU}px;
-          color: ${theme.contentSecondary};
-          ${textStyle('label2')};
-        `}
-      >
-        {label}
-      </h3>
-      {children}
-    </div>
+    <Field
+      label={label}
+      {...props}
+      css={`
+        margin-bottom: 0;
+      `}
+    >
```

Comment:
This change is to inherit spacing certain spacing properties such as when adding a `<Help/>` element to a label as the UI kit already has opinions about that.